### ISSUE TITLE
VM Node Commands Issued By Master Server

### ISSUE THREAD

**Issue by techhelper1** [Feathur/Feathur#115]
The commands should already exist between both sides so all that has to be done is REST/API calls. At least with it being REST support, the end user can use javascript to poll updates from the vm server directly when executing long commands or doing whatever.

It's also best that if the commands being ran (like cat, uname, sed/awk/etc...) they should run under a non-privileged SSH user or the output should be parsed from the local control panel itself via PHP internally or javascript by the end user.

TLDR: The attack surface needs to be reduced when working on something like this.

**Comment by Feathur** [comment]
While entirely true. Changing feathur over to a system like that would take quite a while.

It would be nice to have the module running on the node be faster than SSH at any rate.

**Comment by xaoseric** [comment]
Will look into this with the rewrite I am working on with Laravel
